compiler/transforms/dispatch_regions.py

```diff
@@ -60,46 +60,58 @@ def dispatcher(block: Block, core_cond: builtin.i1, dispatch_rule: Callable):
 
         ## dispatch dm core ops, insert function call
         # in dominator block if changes made
-        func_call_dm = func.Call("snax_is_dm_core", [], [builtin.i1])
+
+        # FIXME: currently assuming that DM core is @ index 1 and compute @ index 0
```

Comment:
unfortunately, this will break my mlperf flow already, maybe for now, an argument for this pass specifying the number of cores in this cluster (defaulting to 2) would be nice? :)

Reply:
compute core can be left at 0 for now, but dm core must be `nb_cores-1`

Reply:
Oh okay!

Reply:
It's there for you to @jorendumoulin !